app/controllers/models_controller.rb

```diff
@@ -84,6 +86,11 @@ def bulk_update
 
   def destroy
     @model.destroy
+
```

Comment:
This would be best in a separate PR, as there are some things to add like a confirmation button, and checking whether or not things *should* be deleted - at the moment I think whenever a model is destroyed, which may happen for various reasons, the filesystem will change. I'd like to default to not removing things unless we're really really sure that's what the user wants :)

Reply:
As mentioned below I can split this out. When I split out the delete functionality, would you be more satisfied with a separate action instead of utilizing the default delete handler to ensure it doesn't have a side effect? If there's another way you think that should be handled I can take a look, although admittedly I'm not the most Rails knowledgeable.